src/api/subscriptions.js

```diff
@@ -337,6 +337,12 @@ export class Subscriptions {
    * @return {?}
    */
   showBestAudienceAction() {}
+
+  /**
+   * Show the meter prompt if the user has free reads.
+   * @return {?}
+   */
+  showMeter() {}
```

Comment:
Should this be exposed on the runtime? My understanding of this surface area is that it is mostly meant for being able to be invoked by publisher code, which most of the swg-basic runtime is pretty fixed to our flows

Reply:
Reverted.